pkg/controller/plan/adapter/openstack/client.go

```diff
@@ -804,11 +804,17 @@ func (r *Client) ensureVmSnapshot(vm *libclient.VM) (ready bool, err error) {
 			"vm", vm.Name, "image", vmSnapshotImage.Name, "imageID", vmSnapshotImage.ID)
 		ready = true
 		if _, ok := vm.Image["id"]; ok {
+			r.Log.Info("the VM is image based, checking the image properties", "vm", vm.Name, "snapshot", vmSnapshotImage.Name)
 			ready, err = r.ensureImageUpToDate(vm, vmSnapshotImage, vmTypeImageBased)
 			if err != nil {
 				r.Log.Error(err, "checking the VM snapshot image properties", "vm", vm.Name, "image", vmSnapshotImage.Name)
 				return
 			}
+
+			if !ready {
+				r.Log.Info("the VM snapshot image properties are not in sync, skipping...", "vm", vm.Name, "image", vmSnapshotImage.Name)
+				return false, nil
```

Comment:
```suggestion
				return
```
?

Reply:
yes, I felt I need to be more explicit here for readability... I don't like naked returns

Reply:
ok, but we have one above and all over this module - so maybe for consistency?

Reply:
ok